boards/arm/nrf21540dk_nrf52840/nrf21540dk_nrf52840.dts

```diff
@@ -95,8 +95,8 @@
 			   <17 0 &gpio1 13 0>,	/* D11 */
 			   <18 0 &gpio1 14 0>,	/* D12 */
 			   <19 0 &gpio1 15 0>,	/* D13 */
-			   <20 0 &gpio0 0 0>,	/* D14 */
-			   <21 0 &gpio0 1 0>;	/* D15 */
+			   <20 0 &gpio0 26 0>,	/* D14 */
+			   <21 0 &gpio0 27 0>;	/* D15 */
```

Comment:
This is reverting a change made in the previous commit.  Either merge the two commits, or remove the change.